source/docs/essentials/add-new-attribute.md

```diff
@@ -0,0 +1,156 @@
+```
+
+In this example, the files we are going to override are located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`, so will create in our `mymodule` folder the following directories:
+
+```bash
+mkdir -p server/modules/acme
+```
+
+(Note that the `acme` name has been chosen arbitrarily.)
+
+### Expand the Product definition
+
+Create a file named `schema.gql` and type in the following code. It means that we are expanding the definition of a product : it can now be rated.
+
+```js
+// mymodule/server/modules/acme/schema.gql
+extend type Product {
+  rate: String
+}
+```
+
+### Implement the resolver
+
+We must now define what Front-Commerce should fetch when `rate` is requested in a GraphQL query ; in other words we must write the code that will **resolve** the queries for the rates.
+
+Create a file named `resolvers.js` and type in the following code. It implements the resolver.
+
+```js
+// mymodule/server/modules/acme/resolvers.js
+export default {
+  Product: {
+    rate: parent => {
+      return parent.rate;
+    }
+  }
+};
+```
+
+### Index our changes
+
+If we want our code to be taken into account when the module is loaded, we must index it.
+
+Create a file named `index.js` and type in the following code. It indexes the schema and the resolver from earlier.
+
+```js
+// mymodule/server/modules/acme/index.js
+import typeDefs from "./schema.gql";
+import resolvers from "./resolvers";
+
+export default {
+  namespace: "Acme",
+  typeDefs: typeDefs,
+  resolvers: resolvers
+};
+```
+
+### Overriding the files
+
+In **this** example, the files to be overridden are `ProductSynthesisFragment.gql` and `Synthesis.js` both located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`. Copy and paste them into your folder.
+
+### Making the query
+
+In `ProductSynthesisFragment;gql`, add the following query line. It means that the application will request the `rate` field as well when sending the query. (Depending on the version of Front-Commerce you are using, the content might differ slightly.)
+
+```diff
+#import "theme/components/organisms/Configurator/ProductConfiguratorFragment.gql"
+#import "theme/modules/AddToCart/ProductStockFragment.gql"
+#import "theme/components/atoms/Typography/Price/ProductPriceFragment.gql"
+#import "theme/modules/ProductView/Details/ProductDetailsFragment.gql"
+
+fragment ProductSynthesisFragment on Product {
+  sku
+  name
+  ...ProductPriceFragment
+  description
++rate
+  ...ProductConfiguratorFragment
+  ...ProductStockFragment
+  ...ProductDetailsFragment
+  configurations {
+    product {
+      sku
+      ...ProductPriceFragment
+    }
+  }
+}
+```
+
+### Displaying the result on screen
+
+We're almost there! In the file `Synthesis.js` (which generates the product page), you can add some code to display the `rate`. Knowing React.js or at least HTML syntax is necessary to perform this step without screwing the other items on the page.
+
+```js
```

Comment:
It would be great to use a `diff` here. It would make things more explicit and easier to follow for the readers.